README.md

```diff
@@ -41,3 +41,44 @@ $ docker start silly_bhaskara
 # Finally re-attach to the running container
 $ docker attach silly_bhaskara
 ```
+
+### Additional Xterm Help
+Once you run the docker run command above and are at the '/home/optee/qemu-optee' 
+command line, you run the `./launch_optee.sh` script and you get an error about 
```

Comment:
What about changing this from
` script and you get an error about not being able to find the xterm (xQuartz) server that looks like this`
to
`script, you might get an error on MacOS saying something about not being able to find the xterm (xQuarts), like this:`

I think that makes it clear that this is a MacOS issue and not normally seen on Linux.